servicenow/README.md

```diff
@@ -0,0 +1,205 @@
+Copyright 2020 Google LLC
```

Comment:
The license header may be put along with the source code instead of README

Reply:
updated both files